### PR TITLE
installed nuxt router

### PR DESCRIPTION
**Pull request by aaronhampe**

### CODE REVIEW

File: frontend/app.vue [hunk 1]
```diff
@@ -1,17 +1,12 @@
+
+
 <template>
-  <div>
-    <NuxtRouteAnnouncer />
-<!--    <NuxtWelcome />-->
-    <TodoList />
+  <div id="app">
+    <NuxtPage/>
+    <NuxtLink to="/">Home</NuxtLink>
```
Comment (ExaCLe) [inline frontend/app.vue]: Do we need this link?

File: frontend/main.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { createApp } from 'vue'
+import App from './App.vue'
+import router from './router' 
+
+console.log("hello")
```
Comment (ExaCLe) [inline frontend/main.js]: remove console.log

File: frontend/main.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+const app = createApp(App)
+
+app.use(router)
+app.mount('#app')
```
Comment (ExaCLe) [inline frontend/main.js]: Do we need this file at all?

**Review by ExaCLe — CHANGES_REQUESTED** [review]

**Comment by aaronhampe** [comment]
ahh yeah thats right

**Review by ExaCLe — APPROVED** [review]

**Pull request merged**